src/components/AuthenticatedAppHeader/SightingsButton.jsx

```diff
@@ -58,18 +64,40 @@ export default function SightingsButton() {
             {mappableSearchResults.map(sighting => {
               const sightingGuid = sighting?.guid;
 
-              // Fake text for now since not enough information is coming back
-              const displayString = sightingGuid;
-              const avatarLetter = displayString[0].toUpperCase();
+              const ownerName = get(
+                sighting,
+                ['owners', 0, 'full_name'],
+                'Unknown User',
+              );
+              const regionLabel = sighting?.locationId_value;
+              const createdDate = formatDate(sighting?.created, true);
+              const sightingDate = formatSpecifiedTime(
+                sighting?.time,
+                sighting?.timeSpecificity,
+              );
+              const avatarLetter = ownerName[0].toUpperCase();
 
               return (
                 <SearchResult
                   key={sightingGuid}
                   avatarLetter={avatarLetter}
                   href={`/sightings/${sightingGuid}`}
                   onClick={closePopover}
-                  primaryText={displayString}
-                  secondaryText="Created on May 4, 2002"
+                  primaryText={
+                    <FormattedMessage
+                      values={{
+                        region: regionLabel,
+                        date: sightingDate,
+                      }}
+                      id="SIGHTING_SEARCH_RESULT_PRIMARY_TEXT"
+                    />
+                  }
+                  secondaryText={
+                    <FormattedMessage
+                      values={{ name: ownerName, date: createdDate }}
```

Comment:
createdDate has fancy formatting, while specified time has yyyy-MM-dd formatting.

Reply:
I think that's fine. Maybe later we can introduce fancy formatting for specifiedTimes